src/index.js

```diff
@@ -366,8 +366,9 @@ export default class AtlasTracking {
         context.events = events || null;
         utils.transmit(action, category, mandatories, user, context, {
             'action': {
-                'location': obj.location !== void 0 ? obj.location : defaults.url,
+                'location': obj.location ? obj.location : undefined,
```

Comment:
This can be written as `'location': obj.location || undefined` like as the lines below.